src/TypesAndInterfaces/config-sync/input-config-sync/profile-field-config.ts

```diff
@@ -101,7 +112,7 @@ export const EDIT_PROFILE_FIELDS_ADMIN:InputField[] = [
     ...EDIT_PROFILE_FIELDS,
```

Comment:
Github won't let me comment on line `#104`  Let's update redux to support 10 partners, since `Range_Slider` is inclusive.

```
new InputRangeField({title: 'Max Partners', field: 'maxPartners', required: true, minValue: 0, maxValue: 10, type: InputType.RANGE_SLIDER, validationRegex: new RegExp(/[0-9]+/), validationMessage: 'Required, between 0-10.'}),
```